src/shared/components/plots/PlotsTab.tsx

```diff
@@ -3679,7 +3679,9 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
                     this.vertGenericAssayOptions.result,
                     selectedEntities,
                     this._vertGenericAssaySearchText,
-                    this.props.hugoGeneSymbols,
+                    Array.isArray(this.props.hugoGeneSymbols)
+                        ? this.props.hugoGeneSymbols
+                        : this.props.hugoGeneSymbols.result,
```

Comment:
seems strange.  i guess this is due to fact that it's used differently in different places. but there should be a better way to handle this.